### ISSUE TITLE
WITH disappearing

### ISSUE THREAD

**Issue by reiniiriarios** [scriptcoded/sql-highlight#17]
**Describe the bug**
Part of queries are disappearing that begin with WITH.

**To Reproduce**
```sql
WITH t1 AS (SELECT
	data_point
	FROM tablename
)
SELECT data_point FROM t1;
```
```js
  let sqlHighlighted = highlight(sql, {
    html: true
  });
```

**Screenshots**
VS Code:
![image](https://user-images.githubusercontent.com/80431903/111053591-0f4f8400-8433-11eb-8943-05c6a64ad84b.png)
highlighted in electron app:
![image](https://user-images.githubusercontent.com/80431903/111053587-01016800-8433-11eb-9588-d9896dff9517.png)

**Node.js**
 - v14.16.0
 - electron v12.0.1

**Browser**
 - electron / Chromium

**Comment by scriptcoded** [comment]
Thanks for the bug report!

I've got a lot to do as it stands, but I'll try and squeeze this in asap.

**Comment by scriptcoded** [comment]
The issue with `WITH` not being included should be resolved in version 3.3.3. The issue with the content being truncated is not resolved though, so I'll leave this issue open for now.

**Comment by scriptcoded** [comment]
Hi!

It would appear that these issues have been fixed by 3.3.3 and 3.3.4. I'll close the issue, but feel free to leave a comment if the issues persist!
